### DISCUSSION TITLE
Localization in Hypermodern Python ?

### DISCUSSION THREAD

**Discussion by oncleben31** [cjolowicz/cookiecutter-hypermodern-python#665] [Ideas]
Do you plan to address the "localization in Python" topic ?

Could be with best practices in code, in CLI and documentation.

**Comment by cjolowicz** [comment]
That sounds interesting! Are you talking about the blog or the project template? There's not much the template can do out of the box, right?

**Comment by oncleben31** [comment]
Both.

I  think the topic it's quite open. It could be:

- How to maintain a project documentation translated with ReadTheDocs.
- Standards or best practices to have the inline documentation and the terminal outputs of a CLI application localized.
- External tool and workflow to maintain translations (example Lokalise)
- Best practices for having Python application **i18n** ready or compliance (date, units, unicode, etc..)

I'm conscious it could be very advanced and not consistent with the goal "_Focus on simplicity and minimalism_". But perhaps some topics could be processed or introduced.
